shark/examples/shark_inference/stable_diffusion/stable_args.py

```diff
@@ -223,4 +229,28 @@
     help="flag for removing the pregress bar animation during image generation",
 )
 
+##############################################################################
+### SD model auto-annotation flags
+##############################################################################
+
+p.add_argument(
+    "--config_path",
+    type=path_expand,
+    default="best_configs.json",
+    help="Path where stores the op config file",
+)
+p.add_argument(
+    "--output_dir",
+    type=path_expand,
+    default="",
+    help="Directory to save the annotated mlir file",
+)
+
+p.add_argument(
+    "--model",
+    type=str,
+    default="unet",
+    help="Options are unet and vae.",
+)
```

Comment:
sd_annotation.py also needs arguments (such as --variant, --version, --max_length, etc) from stable_args.py. I put new arguments here so that when someone runs python sd_annotation.py --help, it shows all flags and these flags are from the same source file.

Reply:
Ah makes sense. Ignore this comment then.